src/box/txn.h

```diff
@@ -885,6 +889,12 @@ txn_is_nop(const struct txn *txn)
 	return txn->n_new_rows + txn->n_applier_rows == 0;
 }
 
+static inline bool
+txn_is_remote(const struct txn *txn)
```

Comment:
Nit: maybe call it `txn_is_fully_remote`? (analogy with `txn_is_fully_local`).

Reply:
Okay, done.